transform_broadcaster:
  ros__parameters:
    refresh_rate: 1000  # Hz